src/pages/using-spark/guides/writing-meaningful-web-content-with-semantic-html.mdx

```diff
@@ -0,0 +1,250 @@
+- **It's good for SEO**. It makes documents more findable, search engines give more importance to keywords inside the content structure.
+
+<span id="table-of-contents" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h2"
+  additionalClasses="sprk-u-mbl sprk-u-Measure"
+  variant="displayTwo"
+>
+  Table of Contents
+</SprkHeading>
+
+- [HTML Elements](#html-elements)
+  - [Creating Elements Using Tags](#creating-elements-using-tags)
+  - [Common HTML Elements For UX writers](#common-html-elements-for-ux-writers)
+- [Semantic Elements](#semantic-elements)
+  - [The Strong Element](#the-strong-element)
+    - [The Strong vs The Bold Element vs Styling Text](#the-strong-vs-the-bold-element-vs-styling-text)
+  - [The Emphasis Element](#the-emphasis-element)
+    - [The Emphasis vs Italic or Idiomatic Text Element](#the-emphasis-vs-the-idiomatic-text-element)
+- [Accessibility](#accessibility)
+
+<span id="html-elements" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h2"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayTwo"
+>
+  HTML Elements
+</SprkHeading>
+
+HTML (Hyper Text Markup Language) consists of series of elements, which enclose, or wrap pieces of content they appear or act a certain way.
+
+Elements describe or represent the meaning, structure, or style of a word, to:
+
+- Web browsers
+- Developers
+- Users of assistive technologies
+
+> For more information, visit <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element"> Modzilla's Developer Documentation - HTML elements</a>.
+
+As a UX Writer, it's important to:
+
+- **Understand** what is the purpose of the enclosed text.
+- **Know** what element to use to use depending on the purpose of the text.
+- **Ensure** we are communicating the intended meaning of the enclosed text to the users.
+
+<span id="creating-elements-using-tags" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h4"
+  additionalClasses="sprk-u-mbs sprk-u-Measure"
+  variant="displayFour"
+>
+  Creating Elements Using Tags
+</SprkHeading>
+
+Use **tags** to create an element. Tags delimit the start and end of elements in the markup.
+
+An element consists of three parts:
+
+- The **opening tag** that represents the start of the paragraph element.
+- The **content** enclosed inside the tags.
+- The **closing tag** that represents where the element ends.
+
+<span id="examples" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h5"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayFive"
+>
+  Example
+</SprkHeading>
+
+`<p>` Download the Rocket Homes mobile application on your mobile device to start looking for your dream home.`</p>`
+
+- The `<p>` represents the start of the paragraph element.
+- The content is the enclosed text (<i>"Download the Rocket.... " </i>) inside the tags.
+- The `</p>` represents where the element ends.
```

Comment:
**Lines 85-115**
I think we could even get rid of this whole section. We do not need to explain what HTML is to the reader. They need to learn the most important part....which is how it affects them as a content writer.

Just something to consider. If I was reading it, I would have abandoned it by now because I'm trying to get to the relevant stuff.

Reply:
We can find the most relevant stuff in this section and put it in `Common HTML Elements for UX Writers`